litmus-portal/frontend/src/components/AccountsTab/PersonalDetails/index.tsx

```diff
@@ -17,89 +15,35 @@ const PersonalDetails: React.FC = () => {
   const handleClose = () => {
     setOpen(false);
   };
-
   const [email, setEmail] = React.useState<string>('');
+  const [userName, setUserName] = React.useState<string>('');
```

Comment:
can you combine them into a single state object with user details? and use one change handler function to update the state with the target value.